make.go

```diff
@@ -52,7 +53,7 @@ var haveSQLite = checkHaveSQLite()
 
 var (
 	embedResources   = flag.Bool("embed_static", true, "Whether to embed resources needed by the UI such as images, css, and javascript.")
-	sqlFlag          = flag.String("sqlite", "false", "Whether you want SQLite in your build: true, false, or auto.")
+	sqlFlag          = flag.String("sqlite", "false", "DEPRECATED. Whether you want SQLite in your build: true, false, or auto.")
```

Comment:
clarify more what this means now. Is this flag ignored, always true, etc?

Reply:
Always true.
Shall we just remove this, or formulate this differently?

Are the supported [targets](https://pkg.go.dev/modernc.org/sqlite#hdr-Supported_platforms_and_architectures) enough for Perkeep? If no, then we may have to keep this machinery for non-cgo-free-sqlite platforms...

Reply:
Let's just remove the flag. Those targets look fine.